src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -185,6 +185,46 @@ public void testFulfilmentRequestByPostFailure_productNotFound() throws Exceptio
     }
   }
 
+  @Test
+  public void testFulfilmentRequestByPostFailure_handDeliveryOnly() throws Exception {
+
+    Mockito.clearInvocations(publisher);
+
+    // Build results to be returned from search
+    CaseContainerDTO caseFromCaseService =
+        FixtureHelper.loadClassFixtures(CaseContainerDTO[].class).get(0);
+    Mockito.when(caseServiceClient.getCaseById(eq(uuid), any())).thenReturn(caseFromCaseService);
+
+    caseFromCaseService.setCaseType("SPG");
+    caseFromCaseService.setHandDelivery(true);
```

Comment:
Minor, but I think that these 2 setup lines should be higher up: inbetween the cases initial assignment and the mockito setup. The gap between the 2 sections implies a change in whats happening but there isn't really. 
It also seems wrong to give the object to Mockito but to then keep fiddling with it.

Reply:
yes, that makes sense, I have changed it now